app-hitl/hitl/src/hitl/commands/run_hitl.py

```diff
@@ -73,10 +79,11 @@ def run_hitl(job_id):
         task_grid=task_grid_gdf,
         output_dir=output_location,
         last_output_dir=last_output_dir,
-        train_kw=dict(
-            num_epochs=5, chip_sz=256, img_sz=256, external_model=True),
-        predict_kw=dict(chip_sz=256, stride=200, denoise_radius=32))
-    logger.info(f"Task grid with priority scores... {task_grid_with_scores.to_json()}")
+        train_kw=PARAMS.get('train', {}),
```

Comment:
How about we use the previous settings dict or a dict with some "rule of thumb" values as the fallback default here instead of an empty one?

Reply:
I have fallback values in the places where the these values are consumed. It amounts to the same thing.